File: db/students.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Text;
namespace Project
{
    #region Students
    public class Students
    {
        #region Member Variables
        protected int _id;
        protected string _first_name;
        protected string _last_name;
        protected int _phone;
        protected string _email;
        protected string _password;
        #endregion
        #region Constructors
        public Students() { }
        public Students(string first_name, string last_name, int phone, string email, string password)
        {
            this._first_name=first_name;
            this._last_name=last_name;
            this._phone=phone;
            this._email=email;
            this._password=password;
        }
        #endregion
        #region Public Properties
        public virtual int Id
        {
            get {return _id;}
            set {_id=value;}
        }
        public virtual string First_name
        {
            get {return _first_name;}
            set {_first_name=value;}
        }
        public virtual string Last_name
        {
            get {return _last_name;}
            set {_last_name=value;}
        }
        public virtual int Phone
        {
            get {return _phone;}
            set {_phone=value;}
        }
        public virtual string Email
        {
            get {return _email;}
            set {_email=value;}
        }
        public virtual string Password
        {
            get {return _password;}
            set {_password=value;}
        }
        #endregion
    }
    #endregion
}

File: students.php
<html>
<head>
<link rel="stylesheet" href="css/table_style.css?v=<?php echo time(); ?>">
<link rel="stylesheet" href="css/checkbox_style.css?v=<?php echo time(); ?>">
</head>
<body>
<?php
$servername = "localhost";
$username = "root";
$password = "";
$dbname = "project";

// Create connection
$conn = new mysqli($servername, $username, $password, $dbname);
// Check connection
if ($conn->connect_error) {
  die("Connection failed: " . $conn->connect_error);
}

$sql = "SELECT id, first_name, last_name, phone, email FROM students";
$result = $conn->query($sql);

if ($result->num_rows > 0) {
  echo "<table class='table' width=100% height=100%><tr><th>ID</th><th>Name</th><th>email</th><th>phone</th></tr>";
  // output data of each row
  while($row = $result->fetch_assoc()) {
    echo "<tr>
	<td>".$row["id"]."</td>
	<td>".$row["first_name"]." ".$row["last_name"]."</td>
	<td>".$row["email"]."</td>
	<td>".$row["phone"]."</td>
	</tr>";
  }
  echo "</table>";
} else {
  echo "0 results";
}
$conn->close();
?>
















</body>











</html>